products/drupal.md

```diff
@@ -31,30 +36,27 @@ releases:
     release: 2020-06-03
     support: 2020-12-01
     eol:     2021-11-02
-    latest:  "8.9.19"
-    lts: true
-  - releaseCycle: "8.8"
```

Comment:
You are removing 8.8.x completely with this pr  and we are keeping older version info for archive purposes. ( Until upstream start to mention them with just major version numbers like 8,x )

So could you fix/addback this part ? other than this everything looks ok.
Thank you for your contribution 🎉